stable/spinnaker/config/spinnaker.yml

```diff
@@ -234,22 +234,9 @@ providers:
       project:
       jsonPath:
 
-  cf:
```

Comment:
the cloud foundry providers is no longer officially supported so removing this from spinnaker.yml